<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGEPaginator</class>
 <widget class="QWidget" name="QtGEPaginator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>45</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtGEPaginator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="firstBtn">
       <property name="text">
        <string>first</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevBtn">
       <property name="text">
        <string>prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pageLbl">
       <property name="text">
        <string>page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pageTxt">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ofLbl">
       <property name="text">
        <string>of</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="countTxt">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lastBtn">
       <property name="text">
        <string>last</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="onPageLbl">
       <property name="text">
        <string>on page:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="onPageList"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
